skripchenkoveta/hometask-6/my_hash_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require './myhash'
+require 'rspec'
+
+# rubocop:disable Metrics/BlockLength
+RSpec.describe MyHash do
+  let(:my_hash) { MyHash.new }
+
+  before do
+    my_hash['li'] = 'za'
+    my_hash['my'] = 13
+    my_hash[:b] = 'name'
+    my_hash[404] = 'error'
+  end
+
+  describe 'describe #initialize' do
+    it 'assigning values ​​to keys' do
+      expect(my_hash['li']).to eq 'za'
+      expect(my_hash['my']).to eq 13
+      expect(my_hash[:b]).to eq 'name'
+      expect(my_hash[404]).to eq 'error'
+    end
+  end
+
+  describe 'describe #delete' do
+    it 'removing item from the hash' do
+      expect(my_hash.delete(404)).to eq []
+    end
+  end
+
+  describe 'describe #length' do
+    it 'number of elements in the hash' do
+      expect(my_hash.length).to eq 4
+    end
+  end
+
+  describe 'negative tests' do
```

Comment:
I would rather organize specs in a way that each method `describe` block contains both positive and negative cases